app/permissions/decidim/challenges/admin/permissions.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Challenges
+    module Admin
+      class Permissions < Decidim::DefaultPermissions
+        def permissions
+          # The public part needs to be implemented yet
+          return permission_action if permission_action.scope != :admin
+
+          allow! if permission_action.subject == :challenges && permission_action.action == :read && read_permission_action?
```

Comment:
why `&& permission_action.action == :read && read_permission_action?` both are checking the same?